packages/react-server-cli/README.md

```diff
@@ -115,6 +115,12 @@ Sets the severity level for the logs being reported. Values are, in ascending or
 
 Default is **'debug'** in development mode and **'notice'** in production.
 
+#### --global-react
+
+Expose `React` as a global variable so modules that use JSX don't have to import it (without obviously using it).
+
+Defaults to **false**.
+
```

Comment:
@aickin - This is something that the old `index.js` in this package provided that `react-server-cli` didn't.

Does this new option look okay to you?